tests/general/LicenseDetectionBearTest.py

```diff
@@ -0,0 +1,66 @@
+from queue import Queue
+import os
+
+from bears.general.LicenseDetectionBear import LicenseDetectionBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensedetect_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as fl:
+        contents = fl.read()
+
+    return contents
+
+
+@generate_skip_decorator(LicenseDetectionBear)
+class LicenseDetectionBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseDetectionBear(Section('name'), Queue())
+
+    def test_license(self):
+        test_file_path = get_testfile_path('apache_license_2.py')
+        file_contents = [load_testfile(test_file_path)]
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseDetectionBear',
+                                test_file_path + ': Apache (v2.0)',
```

Comment:
This is not helpful.  coala is a checker, not a printer.  The user should be able to add  `license=Apache (v2.0)` to `.coafile` and the bear will verify these files use that copyright.